UnityProject/Assets/Scripts/Messages/Server/SpritesMessages/SpriteUpdateMessage.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using Google.Protobuf.WellKnownTypes;
```

Comment:
owo wats this

Reply:
I think it got confused with the different type rider did